packages/core/README.md

```diff
@@ -128,13 +128,17 @@ An object that allows for customization of the Connect Modal and accepts the typ
 
 ```typescript
 type ConnectModalOptions = {
-  showSidebar?: boolean
   showSidebar?: boolean
   /**
    * Disabled close of the connect modal with background click and
    * hides the close button forcing an action from the connect modal
    */
   disableClose?: boolean // defaults to false
+  /**If set to true, the last connected wallet will store in local storage.
```

Comment:
Sweet! We could add these updates to the docs site core module too

Reply:
Yep docs update will come next :)